sklearn/ensemble/weight_boosting.py

```diff
@@ -353,6 +353,24 @@ class AdaBoostClassifier(BaseWeightBoosting, ClassifierMixin):
     feature_importances_ : array of shape = [n_features]
         The feature importances if supported by the ``base_estimator``.
 
+    Examples
+    --------
+    >>> from sklearn.ensemble import AdaBoostClassifier
+    >>> from sklearn.datasets import make_classification
+    >>> X, y = make_classification(n_samples=1000, n_features=4,
+    ...                            n_informative=2, n_redundant=0,
+    ...                            random_state=0, shuffle=False)
+    >>> clf = AdaBoostClassifier(n_estimators=100, random_state=0)
+    >>> clf.fit(X, y)  # doctest: +NORMALIZE_WHITESPACE
+    AdaBoostClassifier(algorithm='SAMME.R', base_estimator=None,
+            learning_rate=1.0, n_estimators=100, random_state=0)
+    >>> print(clf.feature_importances_)
```

Comment:
you can just do `clf.feature_importances_` instead of `print`.

Reply:
same for other prints.

Reply:
Done